### PR TITLE
Don't interpolate when the time fraction matches a keyframe offset

### PR DESCRIPTION
**Pull request by dstoc**

### CODE REVIEW

File: test/testcases/auto-test-matrix-transforms-checks.js [hunk 1]
```diff
@@ -1,65 +1,56 @@
...
-    assert_styles(
-      '.anim',
-      [{'transform':'matrix(1, 0, 0, 1, 100, 0)'},
-       {'transform':'matrix(1, 0, 0, 1, 100, 0)'},
-       {'transform':'matrix(1, 0, 0, 1, 100, 0)'},
-       {'transform':'matrix(0.7071, -0.7071, 0.7071, 0.7071, 100, 0)'},
-       {'transform':'matrix(0.4698, -0.171, 0.171, 0.4698, 100, 0)'}]);
-    }, "Autogenerated");
-  at(2.4, function() {
-    assert_styles(
```
Comment (mithro) [inline test/testcases/auto-test-matrix-transforms-checks.js:57]: You lost 2.4, is that deliberate?

**Comment by mithro** [comment]
Commit 4e37138 "Make interpolate sane at f=0 and f=1" could probably have a better message

File: test/testcases/auto-test-matrix-transforms-checks.js (hunk 1 above)
Comment (dstoc) [inline test/testcases/auto-test-matrix-transforms-checks.js:57]: Not deliberate. I just told the generator to use an interval of 0.4. I'm not sure how to specify how long to run.

But the test at 2.4 shouldn't matter. The animation only runs until t=2

**Comment by dstoc** [comment]
> Commit 4e37138 "Make interpolate sane at f=0 and f=1" could probably have a better message

Done

**Comment by mithro** [comment]
LGTM.

**Pull request merged**